source/documentation/tools.md

```diff
@@ -30,14 +30,14 @@ There are a number of tools available for both R and Python to support package m
 
 For R:
 
-* **conda** - Recommended for R in Analytical Platform. (Conda is only provided on AP with RStudio images which have R 3.5.1 and later. So if you have the older RStudio with R 3.4 then you'll need to use Packrat instead.)
-* **packrat** - not recommended for R, but remains an option.
+* [**conda**](#conda) - Recommended for R in Analytical Platform. (Conda is only provided on AP with RStudio images which have R 3.5.1 and later. So if you have the older RStudio with R 3.4 then you'll need to use Packrat instead.)
+* [**packrat**](#packrat) - not recommended for R, but remains an option.
 
 For Python:
 
-* **venv** and **pip** - Recommended for Python in Analytical Platform, because it's easier and more reliable than conda.
-* **conda environment** installing packages with just **conda** - not recommended, but it might help when you have a package package with a C extension that pip struggles with compiling or installing the binaries
-* **conda environment** installing packages with **conda** and **pip** - not recommended, but gives you the broadest range of package install options. However conda and pip don't play well together - use at your own risk!
+* [**venv** and **pip**](#venv-and-pip) - Recommended for Python in Analytical Platform, because it's easier and more reliable than conda.
+* [**conda environment**](#conda) installing packages with just **conda** - not recommended, but it might help when you have a package package with a C extension that pip struggles with compiling or installing the binaries
```

Comment:
Might be worth being explicit about how `pip` does binary things (so they have a word to Google with). How about:

```
* [**conda environment**](#conda) installing packages with just **conda** - not recommended, but it might help when you have a package package with a C extension that doesn't have a pre-compiled `wheel` package which `pip` can easily use.
```

Reply:
👍 Nice point - added